domain/src/test/java/com/teamagam/gimelgimel/domain/dynamicLayers/remote/SendRemoteAddDynamicEntityRequestInteractorTest.java

```diff
@@ -50,15 +49,7 @@ public void whenExecuted_shouldCallRemoteSourceWithEntity() throws Exception {
     mTestSubject.execute();
 
     //Assert
-    verify(mRemoteSourceMock).addEntity(any(), eq(mGeoEntityMock));
+    verify(mRemoteSourceMock).addEntity(eq(mSingleDynamicLayer), eq(mGeoEntityMock));
```

Comment:
the `eq` matcher can replace the `ArgumentCaptor` for the purpose we used it? Or I got this wrong?